PRFCS/prfc-2.md

```diff
@@ -150,7 +154,7 @@ fn set_spender(token_id: TokenID, spender_address: PublicAddress)
 Gives the account identified by `spender_address` the right to transfer the token identified by `token_id` on behalf of its owner.
```

Comment:
Can do "**Grants** the account identified by `spender` the right to transfer the token identified by `token_id` on behalf of its owner."